lib/naver/searchad/api/core/base_service.rb

```diff
@@ -1,3 +1,11 @@
+require 'addressable/uri'
+require 'addressable/template'
+require 'httpclient'
+require 'naver/searchad/api/options'
```

Comment:
Using absolute path for require across all files is more clear to see what've been loaded. Prefer to use `require` if possible, it's more explicit.